### PR TITLE
Correct artifact uploading of dmg and zip files

### PR DESCRIPTION
**Pull request by jvanzyl**

### CODE REVIEW

File: .github/workflows/BuildPR.yml [hunk 1]
```diff
@@ -43,7 +43,11 @@ jobs:
...
         with:
-          name: GitX built by ${{ matrix.xcode }}-${{ matrix.abi }}
-          path: |
-            GitX-${{ matrix.abi }}.dmg
-            GitX-${{ matrix.abi }}.zip
+          name: GitX-${{ matrix.abi }}.dmg
+          path: GitX-${{ matrix.abi }}.dmg
+      - name: Upload artifact
+        uses: actions/upload-artifact@v3
+        if: ${{ success() }}
```
Comment (hannesa2) [inline .github/workflows/BuildPR.yml:50]: Hmm, I'm not sure, if this `if: ${{ success() }}` is related to previous step, and not to the one more previous prepare step. Anyway, we can give it a try

**Comment by hannesa2** [comment]
@jvanzyl How do you like it https://github.com/gitx/gitx/releases/tag/0.21 ?

**Comment by jvanzyl** [comment]
I will test the release artifacts now and report back.

**Comment by jvanzyl** [comment]
@hannesa2 I tried the ZIP distributions on an x86_64 and arm64 Mac and they work fine. I think they are good for releases now.

**Comment by hannesa2** [comment]
Thank you for testing. The **pre-** release was an accident

**Pull request merged**